code/controllers/subsystem/vote.dm

```diff
@@ -128,44 +122,53 @@ SUBSYSTEM_DEF(vote)
 				var/datum/map_config/VM = config.maplist[SHIP_MAP][.]
 				SSmapping.changemap(VM, SHIP_MAP)
 	if(restart)
-		var/active_admins = 0
+		var/active_admins = FALSE
 		for(var/client/C in GLOB.admins)
-			if(!C.is_afk() && check_other_rights(C, R_SERVER, FALSE))
+			if(!C.is_afk() && check_rights_for(C, R_SERVER))
 				active_admins = TRUE
 				break
 		if(!active_admins)
-			SSticker.Reboot("Restart vote successful.", "restart vote")
+			// No delay in case the restart is due to lag
+			SSticker.Reboot("Restart vote successful.", "restart vote", 1)
 		else
 			to_chat(world, "<span style='boltnotice'>Notice:Restart vote will not restart the server automatically because there are active admins on.</span>")
 			message_admins("A restart vote has passed, but there are active admins on with +SERVER, so it has been canceled. If you wish, you may restart the server.")
 
+	return .
```

Comment:
?

Reply:
Missed than one. They are putting it everywhere in tg station, ew